src/lib/account/isLargeAccount.ts

```diff
@@ -0,0 +1,38 @@
+import { IS_LARGE_ACCOUNT_KEY } from "config/localStorage";
+import { useEffect } from "react";
+import { useLocalStorageSerializeKey } from "lib/localStorage";
+import { useIsLargeAccountData } from "domain/synthetics/accountStats/useIsLargeAccountData";
+import { getStorageItem } from "lib/metrics/storage";
+
+let isLargeAccount = getIsLargeAccountStoredValue();
+
+function getIsLargeAccountStoredValue() {
+  return getStorageItem(IS_LARGE_ACCOUNT_KEY, true) === "true";
```

Comment:
oh, sorry, could you not use getStorageItem? 
This is code for early experiments with metrics, when the plan was to store something in localStorage. 

I'm thinking of removing it because the flag behavior is not obvious

maybe just use localStorage.getItem?